posthog/models/web_preaggregated/sql.py

```diff
@@ -112,15 +112,26 @@ def format_team_ids(team_ids):
     return ", ".join(str(team_id) for team_id in team_ids)
 
 
+def get_team_filters(team_ids):
```

Comment:
Should've been on the other PR, just somewhat less repetition but I will still improve when all tables are created